### PR TITLE
Miscellaneous courier polishing

### PR DESCRIPTION
**Pull request by jknapp25**
Changes:

- communicate speed using tagline from capsule site ("Free, same-day prescription delivery")
- show patient delivery address
- initialize pharmacy options using patient address
- give card content a little breathing room

Example:
_Just for display purposes, Capsule isn't available in brooklyn yet_

https://user-images.githubusercontent.com/3934326/231016205-ee9ec555-d4d6-450e-81ac-74cbb001d7ec.mov

### CODE REVIEW

File: apps/patient/src/views/Pharmacy.tsx [hunk 1]
```diff
@@ -99,31 +98,43 @@ export const Pharmacy = () => {
...
-    if (lat && lng && loc) {
-      setLocation(loc)
-      setLatitude(lat)
-      setLongitude(lng)
-
-      const inAustinTX = /Austin.*(?:TX|Texas)/.test(loc)
-      const isMoPed = order?.organization?.id === process.env.REACT_APP_MODERN_PEDIATRICS_ORG_ID
-      if (inAustinTX && isMoPed) {
-        setEnableCourier(true)
+    if (loc && loc !== location) reset()
```
Comment (samkotlove) [inline apps/patient/src/views/Pharmacy.tsx:101]: Could we get some comments explaining some of the less obvious logic?

**Review by samkotlove — APPROVED** [review]

File: apps/patient/src/components/PickupOptions.tsx [hunk 2]
```diff
@@ -11,11 +19,14 @@ export const PickupOptions = ({
   handleShowMore,
   loadingMore,
   showingAllPharmacies,
-  isMobile
+
```
Comment (captDaylight) [inline apps/patient/src/components/PickupOptions.tsx:22]: empty space, do you have prettier on save?

File: apps/patient/src/views/Pharmacy.tsx [hunk 3]
```diff
@@ -99,31 +98,43 @@ export const Pharmacy = () => {
...
-    setLocationModalOpen(false)
   }
 
   const fetchPharmacies = async () => {
     setLoadingMore(true)
 
+    const searchingInAustinTX = /Austin.*(?:TX|Texas)/.test(location)
+    const patientAddressInAustinTX = true
+    const isMoPed = order?.organization?.id === process.env.REACT_APP_MODERN_PEDIATRICS_ORG_ID
+    if (searchingInAustinTX && patientAddressInAustinTX && isMoPed) {
```
Comment (captDaylight) [inline apps/patient/src/views/Pharmacy.tsx:123]: `patientAddressInAustinTX` looks like it will always be true

Comment (jknapp25) [reply apps/patient/src/views/Pharmacy.tsx:123]: ah! lol yeah was using that for testing that at the end

File: apps/patient/src/components/PickupOptions.tsx (hunk 2 above)
Comment (jknapp25) [reply apps/patient/src/components/PickupOptions.tsx:22]: yeah, it'll leave space though

Comment (jknapp25) [reply apps/patient/src/components/PickupOptions.tsx:22]: removed

File: apps/patient/src/views/Pharmacy.tsx (hunk 3 above)
Comment (jknapp25) [reply apps/patient/src/views/Pharmacy.tsx]: fixed

**Pull request merged**